site/docs/voice/rateLimits.mdx

```diff
@@ -19,4 +19,6 @@ If a rate limit is hit during a `<Forward>`, the call will be allowed, but will
 
 If a rate limit is hit during a `<Transfer>`, the error will be sent to `transferCompleteUrl` also informing the reason in the `errorMessage` field.
 
+As an alternative to hard rate limiting for outbound calls, you may have your account manager enable outbound call queueing. When enabled, your outbound calls will be queued and initiated at a specific dequeueing rate, enabling your application to "fire and forget" when creating calls. Your queue size is limited to a maximum of 5 minutes worth of outbound calls at your dequeing rate. Queued calls may not be modified until they are dequeued and placed, but may be removed from your queue on demand.
```

Comment:
Should we add a note that ordering is not guaranteed?

Reply:
yes

Reply:
We're pulling off the queue based on enqueued time right? So for any given priority, it should be FIFO. Or are you saying that multiple calls submitted at approximately the same time might be enqueued in a different order due to server side timing?

Reply:
We are pulling off the queue based on enqueued time for the particular call engine that this call randomly landed on. Calls that randomly landed on different call engines have no connection and can be sent in any relative order.